sdk/monitor/Azure.Monitor.Query/src/Models/MonitorQueryModelFactory.cs

```diff
@@ -36,5 +36,35 @@ public static LogsQueryResult LogsQueryResult(IReadOnlyList<LogsTable> allTables
             JsonElement errorJson = error.ToObjectFromJson<JsonElement>();
             return new LogsQueryResult(allTables, statisticsJson, visualizationJson, errorJson);
         }
+
+        /// <summary> Initializes a new instance of LogsTableRow. </summary>
+        /// <returns> A new <see cref="Models.LogsTableColumn"/> instance for mocking. </returns>
+        public static LogsTableRow LogsTableRow(IReadOnlyList<LogsTableColumn> columns, Object[] rows)
```

Comment:
C# devs have spent years debating whether to use `Object` vs. `object`, `String` vs. `string`, etc.  Part of that was fueled by Jeff on our team who pushed the .NET class name over the C# keyword in [one of the first major books about .NET](https://learning.oreilly.com/library/view/clr-via-c/9780735668737/).  Luckily, the C# dev lead weighed in with what I now consider the final answer at https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html.  (tl;dr - use `object` instead of `Object`.)

Is `rows` the right name for the parameter since we're creating a single row with this method?  Should it be something more like `values`?

We should also change from `object[]` to `IEnumerable<object>` for `rows`.  We prefer the minimal interface required for input parameters and it'll behave the same because your `JsonElementFromObject` method should still turn it into a JSON array.

I'm less clear on what to do for `columns` because we apparently index it here but only require an `IEnumerable<T>` in the `LogsTable` method below?  It might be worth taking in an `IEnumerable<LogsTableColumn>` for consistency and just calling `ToArray()` on it to make a copy that we can efficiently index.  I'm not super worried about extra allocations during mocking for consistency's sake, but also don't feel strongly if you like the current approach more.

Reply:
I index it here to make a ColumnMap which is passed into the LogsTableRow constructor

Reply:
I'll rename it so it's more clear